spec/models/bot_spec.rb

```diff
@@ -17,4 +17,141 @@
 
   it { should have_valid(:age).when("34") }
   it { should_not have_valid(:age).when(nil, "", 12, 204) }
+
+  let!(:user) { FactoryGirl.create(:user) }
+  let!(:bot) { FactoryGirl.create(:bot, user_id: user.id) }
+
+  before(:each) do
+    Bot.create(
+      name: 'Louisa',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.